src/amo/components/AddonReviewList/index.js

```diff
@@ -265,10 +290,11 @@ export class AddonReviewListBase extends React.Component<Props> {
   }
 }
 
-export function mapStateToProps(
-  state: {| user: UserStateType, reviews: ReviewState |},
-  ownProps: Props,
-) {
+type AppState = {|
```

Comment:
I would not have expected strict type here because the `AppState` looks incomplete to me.

Reply:
It helps to make it strict because it will catch any mistakes within the mapper function. However, we should really define a centralized `AppState` type and reference that in every mapper function.